node/resolvers/list/index.ts

```diff
@@ -91,6 +92,7 @@ const updateItems = async (items: Item[], masterdata: MasterData) => {
 
   const itemsIdAdded = await Promise.all(
     map(
+      // eslint-disable-next-line no-return-await
       async (item: Item) => await addListItem(item, masterdata),
       itemsToBeAdded
     )
```

Comment:
async await, in this case, is redundant 
```suggestion
  const itemsIdAdded = await Promise.all(
    map((item: Item) => addListItem(item, masterdata), itemsToBeAdded)
  )
```